### ISSUE TITLE
Can we expose only one server from Kubernetes publicly?

### ISSUE THREAD

**Issue by labkode** [sciencemesh/sciencemesh#113]
We'll have many services that need to be exposed on the Internet (REVA GRPC gateway, REVA HTTP services, COLLABORA, SWAN, ...)

Should we have a proxy?

**Comment by dagl** [comment]
Yes, we are working with @mateusz-garlacz on ingress and load balancing for the cluster

**Comment by dagl** [comment]
eventually, everything should be accessible on *.cs3mesh.softwaremind.com

**Comment by SamuAlfageme** [comment]
Progress on the default ingress resource for the IOP chart in https://github.com/sciencemesh/charts/pull/4. 

Will evolve the resource in the future to support [name-based virtual hosting](https://kubernetes.io/docs/concepts/services-networking/ingress/#name-based-virtual-hosting) with different `Ingress.spec.rules` per `host`-name
